service/src/main/java/com/ge/predix/acs/attribute/connector/management/AttributeConnectorException.java

```diff
@@ -0,0 +1,18 @@
+package com.ge.predix.acs.attribute.connector.management;
+
+public class AttributeConnectorException extends RuntimeException {
```

Comment:
We are using this as a checked exception, not runtime.  should extend `Exception`